javascriptv3/example_code/dynamodb/scenarios/dynamodb_basics/src/dynamodb_basics.js

```diff
@@ -124,10 +129,8 @@ export const run = async (
       await ddbDocClient.send(new PutCommand(params));
       console.log("Success - single movie added.");
       try {
-        // Before you run this example, download 'movies.json' from https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Js.02.html,
-        // and put it in the same folder as the example.
         // Get the movie data parse to convert into a JSON object.
-        const allMovies = JSON.parse(fs.readFileSync("moviedata.json", "utf8"));
+        const allMovies = JSON.parse(fs.readFileSync("movies.json", "utf8"));
```

Comment:
Can we change the path here to the one in the resources folder instead of having the user move the file from one folder to another?